modules/azure/src/main/java/org/testcontainers/containers/Base64Helper.java

```diff
@@ -0,0 +1,90 @@
+package org.testcontainers.containers;
+
+/**
+ * Taken from jakarta xml bind DatatypeConverter#parseBase64Binary
```

Comment:
I think here we need to adhere to the [Eclipse Distribution License](eclipse.org/org/documents/edl-v10.php) and add the Eclipse copyright notice:
`Copyright (c) 2007, Eclipse Foundation, Inc. and its licensors.`

Also, it might be better to move this to `org.testcontainers.ext` package, like we did for `ScriptUtils`.

Reply:
Base 64 is such a generic thing - can't we use a library rather than having _any_ code for it?
I'd be surprised if we don't already have at least two base64 implementations on the classpath 😂

Reply:
There is built-in Base64 in Java :)

https://www.baeldung.com/java-base64-encode-and-decode

Reply:
The Java implementation has some quirks, but we should definitely try this out first.

Reply:
For some reason I thought that was added after Java 8!

`s/at least two/at least three/` 😆

Reply:
> The Java implementation has some quirks

True.

Actually I tried java version first, but somehow it didn't work. jakarta class was better, but I will give a second chance to java implementation may be we can find another solution.

If can't, will move class to ext package and do proper changes.

Reply:
I think including the code from an Eclipse licensed project might have further consequences we are not aware of and AINAL, so we'd rather not include this. I am optimistic that the Java version can be usable, I think the quirks are some trivial details for our case.

Reply:
Good news, we no longer need that :)